src/services/room-service.js

```diff
@@ -40,33 +33,20 @@ export default class RoomService {
   }
 
   getRoomById(roomId) {
-    return this.roomStore.findOne({ _id: roomId });
+    return this.roomStore.getRoomById({ roomId });
```

Comment:
I always wrapped all arguments of the store functions in an object, as the second param is an object that contains the optional `session` for using it in a transaction. Don't know, I thought this might be a good pattern (similar to how mongodb does the find, update, delete calls. What do you think?

Reply:
Or maybe better, `getRoomById({ roomId, session })` instead of  `getRoomById({ roomId }, { session })`?

Reply:
I honestly would keep the 'mongo' logic inside the store and expose something that suits us best for the store users. To me the store doesn't need to copy the mongo API, I see it as a wrapper.